src/common.ts

```diff
@@ -124,15 +118,13 @@ export async function doActivate(context: ExtensionContext, createLc: CreateLang
       'Vue Semantic Server',
       getDocumentSelector(ServerMode.Semantic),
       getInitializationOptions(ServerMode.Semantic, context),
```

Comment:
For multiple servers, `ServerMode.Semantic` needs to be changed to `ServerMode.PartialSemantic` to tell the language server to disable some features.

Reply:
Thanks! I omitted to check the difference.